### PR TITLE
[#LIEF-7783] IconBadge is missing offset when drawn to Bitmap

### LINKED ISSUE #26: Center drawing in canvas
When rendering the icon badge into the bitmap, the icon is currently top-left aligned but should be centered.

https://github.com/liefery/android-icon-badge/blob/1.3.0/src/main/java/com/liefery/android/icon_badge/IconBadge.java#L176-L178

### PR DESCRIPTION
**Pull request by Empty2k12**
Fixed shadow and badge drawing for exported images.

Closes #26 

screenshot_20171115-171858

### CODE REVIEW

**Comment by Empty2k12** [comment]
Fresh clone and the security error still persists.

File: src/main/java/com/liefery/android/icon_badge/foreground/DrawableForegroundDrawer.java [hunk 1]
```diff
@@ -8,19 +8,33 @@
...
         this.drawable = DrawableCompat.wrap( drawable ).mutate();
     }
 
     @Override
-    public void prepare( int color, int size ) {
+    public void prepare( int color, int size, float elevation ) {
         DrawableCompat.setTint( drawable, color );
-        int offset = (int) Math.round( size * 0.2 );
-        drawable.setBounds( offset, offset, size - offset, size - offset );
+        this.offset = Math.round( size * 0.2 );
```
Comment (taig) [inline src/main/java/com/liefery/android/icon_badge/foreground/DrawableForegroundDrawer.java:24]: What is this magic number?

**Comment by taig** [comment]
Could you please add a section to the bottom of the sample á la "Comparing IconBadgeView with Bitmap" showing Pin, Circle, Square with and without shadow (=> 3x2=6 comparisons) to verify that the Bitmaps render exactly like the views (with minor differences in the shadows of course)

File: src/main/java/com/liefery/android/icon_badge/foreground/DrawableForegroundDrawer.java (hunk 1 above)
Comment (Empty2k12) [reply src/main/java/com/liefery/android/icon_badge/foreground/DrawableForegroundDrawer.java:24]: The icon inset. Basically like a padding.

**Pull request merged**